File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/card/GetCardResponse.java
package com.lattis.ellipse.data.network.model.response.card;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

import java.util.List;

/**
 * Created by ssd3 on 7/26/17.
 */

public class GetCardResponse extends AbstractApiResponse {

    @SerializedName("payload")
    List<GetCardDataResponse> getCardDataResponse;

    public List<GetCardDataResponse> getCardList() {
        return this.getCardDataResponse;
    }

}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/card/GetCardResponse.kt
package com.lattis.data.entity.response.card

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class GetCardResponse : AbstractApiResponse() {

    @SerializedName("payload")
    var cardList: List<GetCardDataResponse>? = null
        internal set

}
